pkg/cmd/server/api/types.go

```diff
@@ -626,6 +627,10 @@ type MasterNetworkConfig struct {
 	IngressIPNetworkCIDR string
 }
 
+type ClusterNetworkEntry struct {
+	ClusterNetworkCIDR string
```

Comment:
+1 on CIDR.
If you plan to do global/local HostSubnetLength field that I was suggesting, then.
```
type ClusterNetworkEntry struct {
      ClusterNetworkCIDR string
      HostSubnetLength     uint32
}
```